### ISSUE TITLE
Custom Dataset question

### ISSUE THREAD

**Issue by cmaclaren** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#332]
Hi there!

Excellent code and am enjoying learning about it. If I want to fine-tune your pre-trained network and identify 18 key points like the normal model, should I train with annotations of only 17 key points? I don't see a need to change the keypoint set up for my goals, but I want to fine-tune just a bit for the data I'm using. So if so, should I annotate with 17 key points or 18?

I began fine-tuning by using the --weights-only flag with just ~250 images to test that are annotated with 18 keypoints. The loss is bouncy, but doesn't appear to be too high compared to what others have show. But when I validate I get an AP and AR of 0 and am having trouble figuring out what is going wrong in training. Is it that I need to make some code changes if it is 18 key points / should I just annotate with 17 and have the model detect the neck in the same manner it does during its original training with coco.

Additionally, I tried visualizing the heatmaps for debugging as they did here https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/issues/235 and all of it looks correct, so I'm not sure what in the training is having issues.

**Comment by Daniil-Osokin** [comment]
Hi! Actually network already predicts 18 keypoints: COCO has 17 + [neck](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/blob/master/datasets/transformations.py#L40). However evaluation with `pycocotools` expects only 17 keypoints, so we do not include neck in the evaluation. Likely your custom 18 keypoints dataset has different keypoints order, than evaluation expects (it expects keypoints in COCO format, can check it [here](https://mmpose.readthedocs.io/en/latest/dataset_zoo/2d_body_keypoint.html#coco)). I suggest to reorder your keypoints to the COCO format, then convert them to training format with [prepare_train_labels.py](https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/blob/master/scripts/prepare_train_labels.py) script and run trainig
